src/js/constants/AppFormErrorMessages.js

```diff
@@ -3,7 +3,10 @@ const ValidConstraints = require("./ValidConstraints");
 const AppFormErrorMessages = {
   appId: [
     "ID must not be empty",
-    "ID cannot contain whitespaces"
+    "Path cannot contain whitespaces",
```

Comment:
Actually let's also make these two lines consistent in using either `cannot` or `must not`. My preference would be `must not`.